README.md

```diff
@@ -2,35 +2,52 @@
 
 Recommended APIs that transport service providers implement, for them to be easily included in the SkedGo platform and [accessible through the TripGo API](https://developer.tripgo.com).
 
-Bike-Sharing:
+## TSP connectors
 
-- Static and real-time data in [GBFS](https://github.com/NABSA/gbfs)
-- [Bike Share High level model](model/BikeShare.md) 
+### Bike-Sharing and Scooter-Sharing:
 
-Moto-Scooter-Sharing:
+- Static and real-time data in [GBFS](https://github.com/NABSA/gbfs) 
+- For GBFS TSP connector, also [provider](shared/#operation/provider) and [coverage](shared/#operation/coverage) endpoints are required.
 
+- [Bike Share High level model](model/BikeShare.md) 
 - [Moto Scooter Share High level model](model/MotoScooterShare.md)
 
-Car-Sharing:
+- (WIP) Specs based on GBFS being re-designed [View in ReDocs UI](micro-mobility).
+
+### Taxi-(like) Services:
+
+- [View in ReDocs UI](taxi).
+
+- (Deprecated) [View in Swagger UI](http://petstore.swagger.io/?url=https://raw.githubusercontent.com/skedgo/TSP-APIs/master/taxi.swagger.yaml). 
+
+### Info:
+
+- [View in ReDocs UI](info).
+
+## Future TSP connectors
+
+These are the current specs, but may change once they are added as TSP connectors
```

Comment:
I don't fully understand this sentence. Does it mean that these are draft specifications that aren't yet fully supported and they might still change?